### PR TITLE
add configurations and views for devise

### PR DESCRIPTION
**Pull request by diegosaul402**
Add configuration for `devise` gem

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+  def index
```
Comment (jclopezdev) [inline app/controllers/home_controller.rb]: Remember to use single line in empty methods 👍

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (jclopezdev) [inline spec/models/user_spec.rb]: Don't commit pending tests files

**Comment by diegosaul402** [comment]
@jclopezdev please review again

**Review by jclopezdev — APPROVED** [review]
LGTM! 👍 😄

**Pull request merged**